components/notebook-controller/config/overlays/openshift/params.env

```diff
@@ -0,0 +1 @@
+odh-kf-notebook-controller-image=quay.io/opendatahub/kubeflow-notebook-controller:1.7-35b81f5
```

Comment:
who (and when) is responsible for updating the image tag?

Reply:
The person that do the release. This procedure take place here for example https://github.com/opendatahub-io/kubeflow/pull/357 

I have open an issue to automate this https://issues.redhat.com/browse/RHOAIENG-9848